.github/workflows/debpack.yml

```diff
@@ -25,8 +25,11 @@ jobs:
       - name: Build Debian Package
         run: debuild --no-tgz-check -us -uc
 
+      # This is moved to reflect the DIY/minimalist aims of the project
+      # Remember to avoid leaking the token in the logs
+      # what I mean is: don't enable verbose on curl
       - name: Generate MD5 and Release the Package
         run: bash .github/release.sh
```

Comment:
why are we going back to this release script?
Didn't we agree on using the github action like everyone else?
We had a really long conversation about it in #18 and now you are ignoring all the things we discussed again...

Reply:
Yeah I saw my comment while reviewing and figured out what I just missed to type: huge gotcha, tricked myself

Another reason is because I've setup a whitelist of actions one can use,
and I think updating actions and the whitelist too is annoying
I would need to want to see their repo, their issues, their documentation, their changes and so on...
I believe the versioned github api will be available for a long time
I see no benefit for using the action **on this project**

- Allowing forks to make their deb, we can fix that, but I don't see how can it be helpful (but I see downsides: partially breaking i3 due to sway support)
- Adding the body atomatically is no issue

- You can continue to make your work using the action, so you're comfortable; I'll cherry pick that

I'm not telling you made a bad choice.

If you have any other concern let me know